src/main/java/net/dv8tion/jda/api/entities/Role.java

```diff
@@ -271,4 +283,51 @@ default RoleAction createCopy()
      */
     @Nonnull
     JDA getJDA();
+
+
+    /**
+     * Enum used to differentiate between the different types of Guild Roles.
+     */
+    enum RoleType
+    {
+        /**
+         * A basic, user-created role.
+         */
+        NORMAL,
+        /**
+         * A role created from a bot on join.
+         */
+        BOT,
+        /**
+         * A role created from a YouTube Member or Twitch Subscriber integration..\
+         */
+        INTEGRATION,
+        /**
+         * The Server Booster role.
```

Comment:
```suggestion
         * The Guild Booster role.
```

Reply:
The role is explicitly called "Server Booster" when it's created. So this should be "The Guild's Server Booster" role

Reply:
In API terminology were always use the term "guild".